src/executable-code/executable-fragment.js

```diff
@@ -313,6 +314,13 @@ export default class ExecutableFragment extends ExecutableCodeTemplate {
       ? document.body
       : this.nodes[0].querySelector(SELECTORS.CANVAS_PLACEHOLDER_OUTPUT);
   }
+    
+  getImportLines() {
+      const allSnippet = this.codemirror.getValue().split("\n");
+      const allImportLines = allSnippet.filter(line => line.indexOf(SEARCH_IMPORT) !== -1);
```

Comment:
As we are using Babel in the project we can freely use all the latest ES spec goodies. For example, there's this cool String method called `includes` that fits our usecase nicely:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes

```suggestion
      const allImportLines = allSnippet.filter(line => line.includes(SEARCH_IMPORT));
```

Reply:
Ok, done